src/app/messages-page/messages-inbox/messages-inbox.component.ts

```diff
@@ -29,14 +42,32 @@ export class MessagesInboxComponent implements OnInit, OnChanges {
     private backendApi: BackendApiService,
     private route: ActivatedRoute,
     private router: Router,
-    private location: Location
+    private appData: GlobalVarsService,
```

Comment:
let's call it `globalVars`, we're trying to get rid of all uses of `appData` in the frontend.